src/test/java/com/github/martapietka/ppm2pbm/ByteReaderTest.java

```diff
@@ -0,0 +1,20 @@
+package com.github.martapietka.ppm2pbm;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+public class ByteReaderTest {
+
+    @Test
+    void convertBytesToInt() throws IOException {
+
+        int expectedInt = 2023;
+        byte[] array = {0x32, 0x30, 0x32, 0x33};
+
+        ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(array);
+
+        Assertions.assertEquals(expectedInt, ByteReader.convertBytesToInt(byteArrayInputStream));
```

Comment:
Tutaj mamy połączoną asercję z wywołaniem testowanej metody, czyli w jednej złożonej instrukcji dzieją się dwie rzeczy naraz. Lepiej takie instrukcje odseparować, a żeby dodatkowo poprawić czytelność testu, proponowałbym rozbić go na części _when_/_then_ (patrz też: https://blog.codecentric.de/given-when-then-in-junit-tests).

No i spróbuj zamiast asercji JUnit używać znacznie popularniejszego dzisiaj AssertJ, tak jak zrobiliśmy to w drugim teście.  

Taki test, rozbity na sekcje _when_/_then_ i wykorzystujący assertJ mógłby wyglądać na przykład tak (żeby wygodnie wywołać `assertThat` tak jak niżej, trzeba będzie wykonać `import static org.assertj.core.api.Assertions.*`):

```java
int expectedResult = 2023;

// when
byte[] array = {0x32, 0x30, 0x32, 0x33};
ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(array);
int result = ByteReader.convertBytesToInt(byteArrayInputStream);

// then
assertThat(result).isEqualTo(expectedResult);